.buildkite/test-template-aws.j2

```diff
@@ -75,7 +75,7 @@ steps:
             - "'cd {{ (step.working_dir or default_working_dir) | safe }} && {{ step.command or (step.commands | join(' && ')) | safe }}'"
             resources:
               limits:
```

Comment:
one more nit comment: is it necessary that we duplicate part of code here? is it possible to reuse the existing code, and only modify say `queue` based on the `step.gpu` value?

Reply:
Our A100 is set up in the EKS cluster so it needs the k8s plugin here to configure the pod (see `podSpec`) in the cluster that runs the job. Other jobs run in a Docker container inside an individual EC2 instance controlled by AWS autoscaling group so it's using Docker plugin.

Reply:
Since the plugins used are different, I don't think we can reuse most of the code here